modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,69 @@
+
+When you create an API Group, you can create different instances of that group with different SLAs and rate limit conditions based on the problem it must solve. You can then publish the API Group to Anypoint Exchange for users to be able to subscribe to use the package.
+
+Because each API Group can include several API instances, and each instance can be included in multiple API Group solutions, package customization options are limitless. 
+
+== How API Groups Works
+
+As an API Group Administrator, you create an API Group for a specific organization and environment. API Groups are versioned, like the APIs, to easily maintain changes.  
+
+You can create instances of an API Group in the same or different environments and add different SLA and rate limit conditions. 
+
+The following diagram explains how you can implement API Groups in different environments.
+image:api-groups-hierarchy.png[]
+
+
+== Use Case
+
+To better understand how API Groups facilitates API consumer engagement, let us look at the example of the Good Weather company. Good Weather is a weather forecasting company that  predicts the atmospheric conditions for a given location and time. 
+
+Good Weather collects quantitative data about the current state of the atmosphere at a given place and using meteorology. Good Weather enables consumers to access their weather forecast data using the following APIs:
+
+* Locations API
+* Forecast API
+* Current Conditions API
+* Indices API
+* Weather Alarms API
+* Alerts API
+* Imagery API
+* Tropical API
+* Translations API
+
+Currently, the consumers access and use the APIs by:
+
+* Scanning the Good Weather company portal to find appropriate APIs to actualize a business flow. The consumers have to determine the API they need by reviewing a long list of APIs on the portal. Additionally, the problem is compounded when developers modify the existing APIs or introduce additional APIs. 
+* Changing multiple APIs to adopt a change to a service plan for each API. This complicates the task of upgrading or downgrading of services.
+* Dealing with access to all the elements of an API when an API access is approved. The API packaging does not restrict access down to the individual call and data elements. This introduces security issues and unnecessary overheads.
+
+== How API Groups Derives Value for Good Weather
+
+Using API Groups, Good Weather can package different APIs based on consumer requirements and scenarios, with specific SLAs that do not clash with the requirements for other consumers. Good Weather can quickly adapt and absorb API changes. With the API Groups: 
+
+* Good Weather can package different sets of APIs to create a business model targeted to different consumers as per their requirements.
+* API consumers can request access to a collection of related APIs instead of individual APIs, which simplifies discovery of assets on the Good Weather portal. 
```

Comment:
```suggestion
* API consumers can access a collection of related APIs instead of individual APIs, simplifying asset discovery on the Good Weather portal. 
```

Reply:
Done! Thanks!